code/game/objects/effects/poster.dm

```diff
@@ -111,6 +111,8 @@
 	var/ruined = FALSE
 	var/random_basetype
 	var/never_random = FALSE // used for the 'random' subclasses.
+	///Exclude posters of this type from being added to the random pool
+	var/random_blacklisted_basetype
```

Comment:
this should be a list var instead for extensibility since it is likely someone will want to filter out multiple different types of posters rather than just one